### PR TITLE
Updated MoodEventCustomList.java, OwnMoodsFragment.java (spinner stuff), activity_main.xml

### PR DESCRIPTION
**Pull request by EveThan**

### CODE REVIEW

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/MoodEventCustomList.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        displayed_emoji.setImageResource(moodEvent.getMood().getIconId());
+
+        Date date = moodEvent.getDate();
+        //set date
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        String dateString = dateFormat.format(date);
+        displayed_date.setText(dateString);
+
+        //set time
+        SimpleDateFormat timeFormat = new SimpleDateFormat("HH:mm");
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/MoodEventCustomList.java]: What does this do? Looks like `timeFormat` isn't used anywhere else?

If you haven't finished this, you probably want to leave a `// TODO:`

**Review by liushuyu — COMMENTED** [review]
Some small nitpicks, otherwise LGTM.

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java [hunk 2]
```diff
@@ -22,36 +23,49 @@
...
 
     /**
      * Mostly a test function to set up the spinner, populate it with a string array from resource.xml
      */
-    private void setUpSpinner(){
-        Spinner spinner = mView.findViewById(R.id.filter_spinner);
+    private void setUpFilterSpinner(){
+        filterSpinner = mView.findViewById(R.id.filter_spinner);
 
         ArrayAdapter<CharSequence> spinnerAdapter = ArrayAdapter.createFromResource(mActivity, R.array.filter,
```
Comment (jenyangk) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java]: There's on error when clicking on the `Filter` bar.
Returns `java.lang.IllegalStateException: ArrayAdapter requires the resource ID to be a TextView`

**Review by jenyangk — DISMISSED** [review]

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java (hunk 2 above)
Comment (LefanTan) [reply app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java]: Outdated implementation, resolved.

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java [hunk 3]
```diff
@@ -29,6 +29,10 @@ protected void onCreate(Bundle savedInstanceState) {
         NavigationUI.setupActionBarWithNavController(this, navController, appBarConfiguration);
         NavigationUI.setupWithNavController(navView, navController);
 
+        //hide action bar
+        ActionBar actionBar = getSupportActionBar();
+        if(actionBar != null)
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java:35]: You can actually hide it in the `styles.xml`

Comment (LefanTan) [reply app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java:35]: tried it, but doesn't work. This is what i had in my styles.

<style name="NoActionBar" parent="@style/AppTheme">
    <item name="android:windowActionBar">false</item>
    <item name="android:windowNoTitle">true</item>
</style>

Comment (liushuyu) [reply app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java:35]: >tried it, but doesn't work. This is what i had in my styles.
>
><style name="NoActionBar" parent="@style/AppTheme"> false true </style>

Well, this is not how it works. See https://github.com/CMPUT301F19T19/legendary-fiesta/pull/33/files#diff-6f870d7e261f41dd3e52a2a129fa1fc5R10-R13 for an example.

**Review by jenyangk — APPROVED** [review]
CustomList and OwnMoodsFragment

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java (hunk 3 above)
Comment (LefanTan) [reply app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java:35]: You can try implementing this method in the future, I'm not quite sure how it works. It uses .setTheme() through the Auth.getInstance() method.

**Pull request merged**